backend/packages/Upgrade/src/api/services/SegmentService.ts

```diff
@@ -73,6 +76,17 @@ export class SegmentService {
     return queryBuilder;
   }
 
+  public async getAllSegmentsAndSubsegments(logger: UpgradeLogger): Promise<Segment[]> {
+    logger.info({ message: `Find all segments` });
```

Comment:
This logger should be different from getAllSegments

Reply:
code updated